resources/assets/less/bem/page-extra.less

```diff
@@ -80,6 +85,12 @@
     width: 100%;
   }
 
+  &__content-overflow-wrapper {
+    max-height: 400px;
```

Comment:
Why use 400px since the max height of _most_ old userpage is 192px?
![](https://puu.sh/zRc7a/cde7c33550.png)

Reply:
From what I see, the userpage doesn't have the option to expand and is locked at 400.
400 is approx the size of an expanded userpage on old web